src/coordinatorapi.cpp

```diff
@@ -129,6 +128,13 @@ restart()
 void
 getCoordHostAndPort(CoordinatorMode mode, string *host, int *port)
 {
+  static bool _firstTime = true;
+  // FIXME:  Could make _cachedHost a 'char[]'.  But then, would
+  //         "*host = _cachedHost;"  replace the string inside *host as needed?
+  //         In particular, if the _cachedHost string object gets destroyed,
+  //         e.g., by a destructor during exit, then this could be a problem.
+  static string _cachedHost;
```

Comment:
Let's not use non-pod static variables. Please convert it to a char array -- `*host = _cachedHost` will continue to work (will invoke copy constructor equivalent).